lib/turbo/engine.rb

```diff
@@ -81,10 +81,13 @@ class Engine < Rails::Engine
     initializer "turbo.test_assertions" do
       ActiveSupport.on_load(:active_support_test_case) do
         require "turbo/test_assertions"
-        require "turbo/broadcastable/test_helper"
 
         include Turbo::TestAssertions
-        include Turbo::Broadcastable::TestHelper
+
+        if defined? ActionCable
+          require "turbo/broadcastable/test_helper"
+          include Turbo::Broadcastable::TestHelper
+        end
```

Comment:
@seanpdoyle what do you think about adding a `ActiveSupport.on_load(:action_cable)` block here? The Rails guides [suggest this technique to execute code that depends on several dependencies](https://edgeapi.rubyonrails.org/classes/ActiveSupport/LazyLoadHooks.html).

Reply:
I like that idea. I was unsure about nesting, but I'll try it out.